libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -111,6 +111,20 @@ static bool _createKeepAliveOperation( const IotMqttNetworkInfo_t * pNetworkInfo
                                        uint16_t keepAliveSeconds,
                                        _mqttConnection_t * pMqttConnection );
 
+/**
+ * @brief Initialize a network connection, creating it if necessary.
+ *
+ * @param[in] pNetworkInfo User-provided network information for the connection
+ * connection.
+ * @param[out] pNetworkConnection On success, the created and/or initialized network connection.
+ * @param[out] pCreatedNewNetworkConnection On success, `true` if a new network connection was created; `false` if an existing one will be used.
```

Comment:
Split this line before merging, it's a bit long, and I don't believe uncrustify does this for you.